src/components/Modal/Modal.module.css

```diff
@@ -0,0 +1,102 @@
+.modalContainer{
+    background-color: rgba(0,0,0,0.5);
+    position: fixed;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
You can use `inset:0` instead of writing all four left, right, top, bottom to 0.